### PR TITLE
Google token

### PR DESCRIPTION
**Pull request by KernTHeAce**

### CODE REVIEW

File: sc_server_auth/config.py [hunk 1]
```diff
@@ -32,6 +32,7 @@ class TokenType(Enum):
     cnt.SQLITE_DB_PATH: "sqlite:///" + "database.db",
     cnt.PRIVATE_KEY_PATH: "private.pem",
     cnt.PUBLIC_KEY_PATH: "public.pem",
+    cnt.GOOGLE_CLIENT_SECRET: "sc_server_auth/client_secret.json",
```
Comment (AlexandrZagorskiy) [inline sc_server_auth/config.py]: Where should I get this secret? I think this argument should be configurable, and also you should provide the ability to pass secret path as an argument of a server running script

File: sc_server_auth/server/auth.py [hunk 2]
```diff
@@ -103,3 +103,23 @@ async def get_access_token(token: models.TokenModel):
         )
     log.debug(f"GetAccessToken response: " + str(response))
     return response
+
+
+@router.post("/get_google_token", response_model=models.GetAccessTokenResponseModel)
+async def get_google_token():
+    flow = InstalledAppFlow.from_client_secrets_file(
+        params[cnt.GOOGLE_CLIENT_SECRET], ["https://www.googleapis.com/auth/userinfo.profile"]
```
Comment (AlexandrZagorskiy) [inline sc_server_auth/server/auth.py]: move this url to constants

File: sc_server_auth/server/auth.py [hunk 3]
```diff
@@ -103,3 +103,23 @@ async def get_access_token(token: models.TokenModel):
...
+    )
+    creds = flow.run_local_server(port=0)
```
Comment (AlexandrZagorskiy) [inline sc_server_auth/server/auth.py:91]: why port always will be 0? Should I change this value sometimes, or not?

File: sc_server_auth/server/models.py [hunk 4]
```diff
@@ -25,6 +39,13 @@ def _validate_token(value):
     return value
 
 
+def _validate_token(value):
+    if len(value) > 800:
```
Comment (AlexandrZagorskiy) [inline sc_server_auth/server/models.py:66]: what is 800? Move it to constants

**Review by AlexandrZagorskiy — CHANGES_REQUESTED** [review]
Let's firstly merge #8, and then you will rebase and refactor your code with new codestyle

**Comment by KernTHeAce** [comment]
Rebase problem: a lot of commits from develop moved to this branch. There is new PR and commits moved to new brunch with cherry-pick

**Pull request closed without merging**